db/schema.rb

```diff
@@ -42,8 +42,8 @@
     t.string "idme_uuid"
     t.string "icn"
     t.string "edipi"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", precision: nil, null: false
```

Comment:
All these `precision: nil` are caused by your local Postgres version I believe, and they should be removed from this PR

Reply:
@ericboehs Have you seen this? I saw this locally on mine as well, is this something platform wants to add to a schema (should be in it's own PR regardless), or would you consider this a misconfiguration on local Postgres version?

Reply:
I could reproduce the schema changes locally as well. looks like there's a pretty [in-depth discussion](https://github.com/rails/rails/issues/44571) associated to this. seems like it may be related to 7.0.1 -> 7.0.2 and it becoming explicit? Also looks like that was referenced in vets-api a few months back. If it were associated to 7.0.2 it surprises me we haven't seen this prior to now 🤔